src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php

```diff
@@ -49,7 +49,16 @@ final class ClassAttributesSeparationFixer extends AbstractFixer implements Conf
      */
     public const SPACING_ONE = 'one';
 
-    private const SUPPORTED_SPACINGS = [self::SPACING_NONE, self::SPACING_ONE];
+    /**
+     * @internal
+     */
+    public const SPACING_ONE_IF_PHPDOC = 'one_if_phpdoc';
```

Comment:
I'm not sure changing `private` to `public` only to use it in tests is a way to go. IMHO using literal strings in tests is perfectly fine.

Reply:
@kubawerlos I'm not sure I followed what you want me to do here. Should I change all of the spacing constants here to `private`, then use the literal strings in the tests?

Reply:
There is no private to public change here though I agree with @kubawerlos: we should use literal strings in tests rather than the constant. Otherwise, we might change the constant and thus introduce a BC break without the test detecting it. Can you make these constants private please?

Reply:
Done